pushy/src/main/java/com/relayrides/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -63,8 +65,10 @@
     private static final String SOUND_KEY = "sound";
     private static final String CATEGORY_KEY = "category";
     private static final String CONTENT_AVAILABLE_KEY = "content-available";
+    private static final String MUTABLE_CONTENT_KEY = "mutable-content";
 
     private static final String ALERT_TITLE_KEY = "title";
+    private static final String ALERT_SUBTITLE_KEY = "subtitle";
```

Comment:
I wonder if there's also a `subtitle-loc-key` and `subtitle-loc-args` field that hasn't been documented. I'll try to test that a bit later today, too.